README.md

```diff
@@ -1,141 +1,146 @@
+### Appeals
 
-Once the dispute is initialized and the jury formed, the adjudication process is broken up into four time windows, the duration of each can be different depending on the specific court being used:
+Each dispute is subject to a maximum number of appeal rounds. Since each dispute and appeal round has a fixed duration, the max appeal rounds also determines the maximum amount of time before a final decision is reached.
 
- - **Evidence Submission**: a period when parties of an agreement can submit statements and evidence to the jury.
- - **Jury Ruling Period**: a period for jurors to review submitted evidence and submit a ruling. The ruling determines if the case should be (1) escalated for review, (2) dismissed, (3) have the Agreement's FINALIZATION parameter updated and/or (3) whether collateral should be reassigned between parties. 
- - **Ruling Review Period**: a period which is triggered if atleast one juror flags the case for review, the evidence and jurors rulings are reviewed by a second set of jurors selected from the Review Court. The review process is intended to punish jurors who do not self-report bribery or vote in such a way that is not coherent with the jurors code of conduct.
- - **Appeal Period**: a period after the court has provided a ruling, but before reputation is assigned to participating jurors.
-### 3.2.1 Jury Ruling
-The Jury Ruling mechanism operates as a schelling point game where jurors individually assess the evidence and make a ruling and have an incentive to rule along with the majority of other jurors.
+Appeals can be triggered after a dispute has been resolved with a preliminary ruling in favor of one outcome or the other. In order for an Appeal to occur both sides of the dispute must deposit additional collateral. If neither side deposits the required collateral to trigger the next appeal round, the preliminary ruling is finalized. If only one side deposits the required collateral the ruling is immediately finalized in their favor. If both sides deposit the required collateral then the appeal round is scheduled.
 
-Since a ruling must aggregate input from all jurors, the following procedure is used to create a jury ruling and reward participants.
+The amount of required collateral depends on the appeal round and is a multiple of the fees required to compensate selected jurors. In higher appeal rounds where more juror stake is selected to adjudicate the dispute, the base amount of collateral required will be higher. The amount of collateral required will be a multiple of this amount, 2x the base amount for the side which is reinforcing the preliminary ruling, and 3x the base amount for side which is appealing the preliminary ruling. These deposit do not need to be supplied by a single party, but can be crowdsourced. When a final ruling is reached, the collateral of the loosing side will be used to compensate jurors and reward funders of the opposing side proportional to their contribution.
```

Comment:
In this section it seems that "collateral" is mixed with fees for jurors, while in the first round (the dispute itself) it says: "they will need to deposit an equivalent amount of collateral, along with initial *dispute fees* determined by the Aragon Court" (so there's a clear distinction between collateral and jurors fees).
Actually from this paragraph one may think that collateral in subsequent appeals is only meant for juror fees, but in `Redistributing Collateral` section below it says:
"If there was any appeal rounds before the final ruling, the total amount of collateral that was deposited for triggering each round will be assigned to the appealer supporting the final ruling." 
All of this made me think, assuming in every round there are both collateral to be slashed/redistributed among proposer and appealer and fees to compensate jurors, that if we route (at contract level) all appeals through Proposal Agreements app (and then Proposal agreements app calls Court to raise the dispute) then we could collateral and fees separate in every round, so collateral would always  stay on Proposal Agreements and Court wouldn't need to know about it. When Court would call `rule` function in Proposal Agreements, collateral would be redistributed. I think this would be cleaner as would keep Court agnostic about external collateral requirements.

Reply:
Besides I wonder if routing appeals through Proposal Agreements could be something useful generally speaking for the Court, in the sense that an organization using the Court may want to control who can appeal (e.g., only shareholders of org, but requiring stake in DAI which anybody can have).
Actually the original [ERC 792 interface](https://github.com/ethereum/EIPs/issues/792) says: "It should be called by the Arbitrable contract."

Reply:
The problem is that it wouldn't be possible for the ruling to be flipped if someone puts up collateral to start an appeal against the previous round's ruling, but no one puts collateral to support the current court ruling.

Apart from this, the collateral for appeals doesn't necessarily need to be the same collateral type as the one used in the Proposal Agreement (which could be an organization's native token for example). By making it the same token as the fee token, the collateral requirement can be proportional to the juror fee (without needing a price feed).

Reply:
> The problem is that it wouldn't be possible for the ruling to be flipped...
Hm, good point, although we could move this logic to Proposal Agreements.
> By making it the same token as the fee token, the collateral requirement can be proportional to the juror fee
But what do you need it to be proportional? I may be still confused regarding collateral and fees, but the way i understand it:
- Collateral: any token the org decides (for instance their native token), from and for proposer and challenger. They put it and they have it slashed if they loose, they earn it if they win. Jurors don't ever touch it.
- Fees: got from ANT, paid by proposer and challenger, earn by jurors. Proposer and challenger never earn it.
That's why I think it's a good thing to have them decoupled, and that the Court is agnostic about collateral (another org consuming Court services may decide not to require collateral at all).

Reply:
> The problem is that it wouldn't be possible for the ruling to be flipped if ...

Actually I think the ruling could still be flipped. Everything would be the same, but the Proposal Agreements contract would keep info about collateral used in appeals, which would be redistributed on `rule` function.